### PR TITLE
Create ebay token constant

### PR DESCRIPTION
**Pull request by topspinj**
Added ebay token to the constants file.

### CODE REVIEW

File: APIs/online-shopping.php [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 	require_once 'inc/connection.inc.php';
 	require_once 'inc/function.inc.php';
+	require_once 'inc/constants.inc.sample';
 
-	$ebay_app_name = 'NSIT22619-c5e7-41ee-9311-cbde5b60ca2';
+	$ebay_app_name = EBAY_TOKEN;
```
Comment (prabhakar267) [inline APIs/online-shopping.php]: You can remove the `ebay_app_name` variable

File: APIs/inc/constants.inc.sample [hunk 2]
```diff
@@ -30,6 +30,6 @@ define("TWITTER_CONSUMER_KEY", "XXXX");
 define("TWITTER_CONSUMER_SECRET", "XXXX");
 define("TWITTER_OAUTH_TOKEN", "XXXX");
 define("TWITTER_OAUTH_TOKEN_SECRET", "XXXX");
-
+define("EBAY_TOKEN", "NSIT22619-c5e7-41ee-9311-cbde5b60ca2");
```
Comment (prabhakar267) [inline APIs/inc/constants.inc.sample]: Remove the key and add "XXXX"

Comment (topspinj) [reply APIs/inc/constants.inc.sample]: Where do you plan to store the token?

**Comment by prabhakar267** [comment]
@topspinj PR looks fine, I have approved it
I will merge it once i get back to home 👍

**Comment by prabhakar267** [comment]
Merged!

**Pull request closed without merging**